packages/wrangler/src/sites.tsx

```diff
@@ -76,21 +80,12 @@ async function createKVNamespaceIfNotAlreadyExisting(
   }
 
   // else we make the namespace
-  // TODO: use an export from ./kv
-  const json = await fetchResult<{ id: string }>(
-    `/accounts/${accountId}/storage/kv/namespaces`,
-    {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify({ title }),
-    }
-  );
+  const id = await createNamespace(accountId, title);
+  console.log(`🌀 Created namespace for Workers Site "${title}"`);
```

Comment:
❤️

Reply:
Given that this console.log does not appear in any of our test snapshots, I guess that means that we are not testing this code path?

Reply:
Urgh, you're right. I'll send a PR with tests specifically for that asap.